betterlockscreen

```diff
@@ -99,7 +102,7 @@ lock() {
 
     local image="$1"
 
-    i3lock \
+    i3lock --nofork \
```

Comment:
@souravdas142 This is not part of the requested changes. #28  seems to indicate that we do want to fork when suspending. `[[ $runsuspend ]] || lockargs+=(-n)` seems to cover this scenario already.

Reply:
you can review now.